packages/react-devtools-shared/src/devtools/store.js

```diff
@@ -141,6 +142,9 @@ export default class Store extends EventEmitter<{|
   _supportsReloadAndProfile: boolean = false;
   _supportsTraceUpdates: boolean = false;
 
+  // Needed for reload and profile
+  _supportsSynchronousXHR: boolean = false;
```

Comment:
Slightly awkward conflict now with the `_supportsReloadAndProfile` variable above. So `_supportsReloadAndProfile` will be true even if `_supportsSynchronousXHR` is false.

Reply:
Maybe the idea of disabling the button with a tooltip isn't worth it in retro and we should just hide it. Seems odd to return `true` for `supportsReloadAndProfile` even if we know the environment doesn't support it. Okay let me make that change.